### ISSUE TITLE
StartTimestamp

### ISSUE THREAD

**Issue by Hiroszek** [Pizzabelly/EasyRP#121]
Is it possible to set the value of "StartTimestamp=" so that the start time is set to the one when the program starts?
(Yes, I know I have to set the unix time there, but I would like it to set itself when I run the program to keep it counting as much as I do.)

**Comment by bhavyagosai** [comment]
YES it is now! Check out the Pull req https://github.com/Pizzabelly/EasyRP/pull/138 or this [Repo!](https://github.com/bBSempai/EasyRP)

Hope it helps :)
